### DISCUSSION TITLE
Dupe with Mystical & Simple Storage

### DISCUSSION THREAD

**Discussion by NordicGamerFE** [ACGaming/UniversalTweaks#506] [Bugfix Suggestions]
https://github.com/NordicGamerFE/Ocean-Outlast/issues/155

**Comment by ACGaming** [comment]
As I plan on working on a Simple Storage Network fork eventually, a fix for this will be part of it if feasible.

**Comment by ACGaming** [comment]
[Lothrazar was recently working on the 1.12.2 version of Simple Storage Network again](https://www.youtube.com/watch?v=o9nfja-Il4I), so this might be something to report to the original mod eventually.

**Comment by NordicGamerFE** [comment]
I posted a issue on simple storage network we will see where it goes from there
